environment.yml

```diff
@@ -20,3 +20,4 @@ dependencies:
   - rapidnj
   - h5py
   - pp-sketchlib
+  - graph-tool
```

Comment:
This also needs to be added to:
- `setup.py`
- `requirements.txt`
- `docs/conf.py`
- `docs/installation.rst`

Reply:
graph-tool version >=2.3.1 specified in all these documents in f6b86f6.